GitCommands/Git/GitModule.cs

```diff
@@ -2830,8 +2830,10 @@ public static string GetSelectedBranchFast(string? repositoryPath, bool setDefau
         /// <summary>
         /// Gets the current branch.
         /// </summary>
-        /// <param name="setDefaultIfEmpty">Return "(no branch)" if detached.</param>
-        /// <returns>Current branch name.</returns>
+        /// <param name="setDefaultIfEmpty">Defines the value returned if HEAD is detached. <see langword="false"/> to return <see cref="string.Empty"/>; <see langword="true"/> to return "(no branch)".</param>
+        /// <returns>
+        /// The name of the branch (for example: "main"); the value requested by <paramref name="setDefaultIfEmpty"/>, if HEAD is detached.
+        /// </returns>
         public string GetSelectedBranch(bool setDefaultIfEmpty)
```

Comment:
Igor requested inverted logic:
```suggestion
        public string GetSelectedBranch(bool emptyIfDetached)
```